holidayextras/brands
# Deleting all versions of thorpe park

#### What does this PR do? (please provide any background)
- Deletes Thorpe dark
- Deletes Derren Brown Promotion
- Deletes Fright Night
- Modify's Thorpe Park to be a dark skin

#### How can this be tested?
`cd pattern-library && npm link ../brands` to either the wall or pattern library (you may need to delete a few bits in PL.. come see me for details)

#### Any tech debt?
None

#### Screenshots / Screencast
NEW
<img width="807" alt="screen shot 2018-04-25 at 09 29 17" src="https://user-images.githubusercontent.com/778942/39234439-2877a3d8-486b-11e8-8eea-34ab9550cbb3.png">
OLD
<img width="807" alt="screen shot 2018-04-25 at 09 29 44" src="https://user-images.githubusercontent.com/778942/39234457-35ff573a-486b-11e8-80c0-7bdef4863ee1.png">


#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
